app/policies/group_policy.rb

```diff
@@ -253,6 +253,25 @@ def destroy_attachment?
     false
   end
 
+  def create_metadata_template?
```

Comment:
Sorry I totally missed this, can pluralize these 3 policy methods and the ones you added to the project_namespace_policy

Reply:
Updated in 69e1a8d